settings.gradle

```diff
@@ -1,3 +1,2 @@
 rootProject.name = 'backend'
 include 'eddnMessageListener'
-include 'modulith'
```

Comment:
why only remove one? shouldn't it be both or none?

Reply:
I didn't look into this as it was outside the modulith, so I assumed it was a separate project

Reply:
ideally we would move these two project to a repository each, so that the monolith can be the top level project